packages/analytics/analytics-utilities/src/types/explore/requests.ts

```diff
@@ -28,18 +28,78 @@ export const queryableRequestDimensions = [
   'service_port',
   'service_protocol',
   'sse',
+  'status_code',
```

Comment:
What does including `status_code` here help with? We expect `status_code` to be a number value in the API, but now it's being included in ways that expect a string filter value like the `RequestEqualsFilter`.

Reply:
Ah nvm I see you had a comment in slack about this and I see now how the dimensions are used below with the local searchable list.

Reply:
It behaves more like a dimension filter in that there's a searchable list of things to select from locally when filtering, in addition to allowing user inputted values. I don't think there's another metric that behaves like this.

I may be wrong, in that case, we should split the metrics into two categories, one for metrics for which we pre-populate a list of things the user can pick from, and ones for which we do not